test/prepare.sh

```diff
@@ -2,18 +2,52 @@
 
 set -eux -o pipefail
 
-IMG="${IMG:-controller:test}"
+IMG="${IMG:-localhost/controller:test}"
 LOGDIR="${LOGDIR:-/tmp/logs}"
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-}"
+CERT_MANAGER_VERSION="${CERT_MANAGER_VERSION:-1.16.1}"
 
 mkdir -p "${LOGDIR}"
 
+if [ -z "${CONTAINER_RUNTIME}" ]; then
```

Comment:
```suggestion
if [[ -z "${CONTAINER_RUNTIME}" ]]; then
```
We're writing bash, so let's use the bash's double brackets.